plugins/hdf/CMakeLists.txt

```diff
@@ -31,11 +31,20 @@ f3d_plugin_declare_reader(
   CUSTOM_CODE "${CMAKE_CURRENT_SOURCE_DIR}/exodus.inl"
 )
 
+f3d_plugin_declare_reader(
+  NAME NetCDF
+  EXTENSIONS nc cdf ncdf
+  MIMETYPES application/netcdf application/x-netcdf
+  VTK_READER vtkNetCDFReader
+  FORMAT_DESCRIPTION "NetCDF"
+  CUSTOM_CODE "${CMAKE_CURRENT_SOURCE_DIR}/netcdf.inl"
+)
+
 f3d_plugin_build(
   NAME hdf
   VERSION 1.0
   DESCRIPTION "VTK HDF formats support"
-  VTK_MODULES IOHDF IOExodus
+  VTK_MODULES IOHDF IOExodus IONetCDF
```

Comment:
actually you need to add this in:

 - .github/actions/vtk-install-dep/action.yml
 - CMakeLists.txt
 - doc/dev/BUILD.md
 -